package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "carousell-backend-bootcamp",
   "version": "1.0.0",
-  "type": "module",
   "scripts": {
     "start": "nodemon index.js"
   },
   "dependencies": {
     "cors": "^2.8.5",
+    "dotenv": "^16.0.1",
     "express": "^4.18.1",
+    "nodemon": "^2.0.19",
```

Comment:
Should that not be installed locally?
Sorry I dont know if you teach it that way or not.